composer.json

```diff
@@ -7,13 +7,19 @@
     "autoload-dev": {
         "psr-4": {
             "App\\Spec\\": "spec/"
-        }
+        },
+        "files": [
+          "src/prodecural.php"
```

Comment:
Sorry about the style snafu here. Without an `.editorconfig` Atom does not want to play nice with current formatting.